cloudsim-plus/src/main/java/org/cloudbus/cloudsim/brokers/DatacenterBroker.java

```diff
@@ -191,7 +205,7 @@
     /**
      * Sends a list of VMs for the broker that their creation inside some Host will be requested just after a given delay.
```

Comment:
Change to "Sends a list of VMs for the broker so that their creation inside some Host will be requested just after a given delay."